### ISSUE TITLE
Feature Request: Request a warning when exiting Studio

### ISSUE THREAD

**Issue by nguyenquanghoang** [Automattic/studio#418]
### What

- There should be a notification: Are you sure you want to quit Studio? All of your sites will be shutdown.
- And there should be a choice: Yes or No

![Are you sure you want to quit Studio](https://github.com/user-attachments/assets/ca805abf-8ad5-4ae6-aa48-8128bb4988b3)

### Why

Sometimes I accidentally click the shutdown button which interrupts the activity and wastes time

**Comment by wojtekn** [comment]
Thanks for the suggestion @nguyenquanghoang, it sounds reasonable to ask users to do so.

**Comment by nguyenquanghoang** [comment]
Yes @wojtekn usually when you are doing something on the Site and have not saved it yet, this will be an extremely useful feature.

**Comment by wojtekn** [comment]
It's done - now, when there are sites running, Studio asks whether the user wants to keep them running, stop them, or cancel the action.

<img src="https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/c2334f77-49bb-4ecd-9063-54a46b2d6421/0857c681-a588-471e-9f4a-daf48a566c68?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi9jMjMzNGY3Ny00OWJiLTRlY2QtOTA2My01NGE0NmIyZDY0MjEvMDg1N2M2ODEtYTU4OC00NzFlLTlmNGEtZGFmNDhhNTY2YzY4IiwiaWF0IjoxNzcwMjE4OTQzLCJleHAiOjE4MDE3ODk1MDN9.J6kDc-U3jFspnM-TrPOKbGrUgTMZM26s4l4Ufz1GIkQ " alt="CleanShot 2026-02-04 at 16.27.12@2x.png" width="520" data-linear-height="664" />
